application/controllers/template_controller.py

```diff
@@ -0,0 +1,64 @@
+from application.models.models import CommunicationTemplate
+from application.utils.session_wrapper import with_db_session
+from application.utils.exceptions import InvalidTemplateException, DatabaseError
+from application.models.schema import template_schema
+from jsonschema import validate, ValidationError
+from structlog import get_logger
+
+
+logger = get_logger()
+
+UPLOAD_SUCCESSFUL = 'The upload was successful'
+INVALID_TEMPLATE = 'Template is invalid'
+PREEXISTING_TEMPLATE = 'Id already exists'
+
+
+def get_template_by_id(template_id, session):
+    return session.query(CommunicationTemplate).filter(CommunicationTemplate.id == template_id).first()
+
+
+def validate_template(template):
+    try:
+        validate(template, template_schema)
+    except ValidationError as exception:
+        logger.exception("Attempted to upload invalid template")
+        raise InvalidTemplateException(INVALID_TEMPLATE, status_code=400)
+
+
+class TemplateController(object):
```

Comment:
Is the class adding much value here? We'd get the name spacing from the package I guess?